### DISCUSSION TITLE
Bought from google pay

### DISCUSSION THREAD

**Discussion by stardust976** [aclap-dev/video-downloadhelper#455] [License & Purchase requests]
### What's your issue?

I bought lifetime license with google pay at 02.09.2019 but now cant license your software. What will i do?

### The 10 first digits (only 10) of your license number or email address

stardust461976@hotmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
I sent you an email.

**Comment by stardust976** [comment]
hi again. when i used paid licence number whick u granted i see "license blocked" how can i solve that?
________________________________

**Reply by jcfrog** [reply]
It's probably because you used the license that has been refunded.
The license to be to used is the one ending with 53359e968a

**Comment by stardust976** [comment]
i bought software over chrome at 2019. it said lifetime. in years it activated auto when i change my pc but at last it didnt activated. i sent mail. and got that license number. i entered but  license blocked...
________________________________

**Reply by jcfrog** [reply]
I sent you an email

**Comment by stardust976** [comment]
Chrome Web Store09 Şubat 2019 Cumartesi 10:35:29 UTCÖğeMiktarFiyatLifetime Premium VDH
Lifetime premium features on Video DownloadHelper1TL 31,79 TRYVAT TL 4,85 TRY
TL 31,79 TRYPayment method
MASTERCARD-6347Google Sipariş Numarası
10370910874874185126.b6e9310d47d34b07a3e2d6328961fb0d

this is the mail only i got after my payment. no license code received...

________________________________

**Comment by stardust976** [comment]
not received an email yet
________________________________

**Reply by jcfrog** [reply]
please send me an email at licenses@downloadhelper.net so that I'm sure I use the good address.
